docs/admin/federation-apiserver.md

```diff
@@ -18,11 +18,6 @@
 If you are using a released version of Kubernetes, you should
 refer to the docs that go with that version.
 
-<!-- TAG RELEASE_LINK, added by the munger automatically -->
-<strong>
-The latest release of this document can be found
-[here](http://releases.k8s.io/release-1.3/docs/admin/federation-apiserver.md).
-
```

Comment:
All the changes to CHANGELOG.md and the other many md files under docs/ look a bit suspicious. Was it intentional?